public/javascript/main.js

```diff
@@ -377,4 +377,16 @@ $(function(){
           $(this).data('chart', chart);
       }
   })
+
+  // Event tracking
+  // Clicks on elements with data-ga-track-click attribute will be recorded
+  function trackClick(eventDescription){
+    eventDescription = eventDescription || 'Undescribed event.';
+    ga('send', 'event', 'user interaction', eventDescription, document.title);
```

Comment:
I think you referenced the wrong nick upthread.

@jacksonj04 Any thoughts on what to put in the event fields? At the moment I have:

  `hitType: 'event',`
  `eventCategory: ‘user interaction',`
  `eventAction: eventDescription,`
  `eventLabel: document.title`